stories/Picker/Picker.stories.js

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { createPicker } from "./Picker";
+
+export default {
+  title: "Picker",
+  argTypes: {
+    scaleMultiplier: {
+      name: "active scale multiplier",
+      defaultValue: 95,
+      control: { type: "range", min: 70, max: 98 },
+    },
+    duration: {
+      name: "active duration (ms)",
+      defaultValue: 80,
+      control: { type: "range", min: 50, max: 200 },
+    },
+    ease: {
+      name: "active ease function",
+      defaultValue: "ease-out",
+      options: ["ease-in-out", "ease-in", "ease-out", "ease-linear"],
+      control: { type: "select" },
+    },
+    animateChevronOnly: {
+      name: "only animate chevron on active state",
+      defaultValue: true,
+      type: { name: "boolean" },
+    },
+    label: { control: "text" },
+    isDisabled: {
```

Comment:
Could you add the option for isQuiet?

Reply:
easy enough: https://github.com/adobe/spectrum-prototypes/pull/15